developers/weaviate/search/basics.md

```diff
@@ -12,21 +12,23 @@ import PyCode from '!!raw-loader!/_includes/code/howto/search.basics.py';
 import PyCodeV3 from '!!raw-loader!/_includes/code/howto/search.basics-v3.py';
 import TSCode from '!!raw-loader!/_includes/code/howto/search.basics.ts';
 
-## Overview
+import ClassToCollectionNote from '/_includes/class-to-collection-transition-note.mdx' ;
 
-This page shows the core concepts on how to perform searches and retrieve objects.
+<ClassToCollectionNote /> 
 
-import ClassToCollection from '/_includes/class-to-collection-transition.mdx' ;
+This page has basic search examples. 
```

Comment:
Let's remove this as it repeats the title.

Reply:
Updated